### ISSUE TITLE
backbone

### ISSUE THREAD

**Issue by lijain** [zhanghang1989/ResNeSt#99]
Hello, I will replace the backbone network of RESNET with your resnest. During the training process, the loss value decreases very slowly (LR here is the same as that of RESNET). Similarly, in 20 epoches, the loss of RESNET is 0.06 and that of resnest is about 0.5. What's going on next?

**Comment by zhanghang1989** [comment]
ResNeSt need to use large batch size per gpu or SyncBatchNorm.

**Comment by Dpw506** [comment]
Hi, What is the process of using RESNETST instead of RENET?

**Comment by zhanghang1989** [comment]
> Hi, What is the process of using RESNETST instead of RENET?

https://github.com/zhanghang1989/ResNeSt#pytorch-models
